Makefile.PL

```diff
@@ -206,7 +206,8 @@ sub ssleay_get_build_opts {
         }
     }
 
-    for ($prefix, "$prefix/lib64", "$prefix/lib", "$prefix/out32dll") {
+    # Note: the order matters for macOS Monterey.
```

Comment:
```suggestion
    # Note: the order matters for macOS Monterey. See GH-329.
```

Might be worth referencing #329 here, in case someone wants to change the order again in future and wonders why Monterey is an edge case.

Reply:
Appveyor shows that it now fails to link with recent Strawberry Perls. I'll work on this more and also add pointers to further information. I'll also add a `Changes` entry.